File: BS_6/Experiments/mkrenz/HelloWorld/app/src/main/java/com/example/helloworld/MainActivity.java
package com.example.helloworld;

import androidx.appcompat.app.AppCompatActivity;
import android.content.Intent;
import android.view.View;
import android.view.View.OnClickListener;
import android.view.Menu;
import android.view.MenuItem;
import android.widget.Button;
import android.os.Bundle;

public class MainActivity extends AppCompatActivity {



    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        Button button = (Button) findViewById(R.id.button);
        button.setOnClickListener(new View.onClickListener());

        @Override
        public void onClick(View v) {

            goToSecondActivity();

        }
    }





    private void goToSecondActivity() {

        Intent intent = new Intent(this, SecondActivity.class);

        startActivity(intent);

    }

}

File: BS_6/Experiments/dals/TrialGameAS/app/src/main/java/com/example/trialgame/MainActivity.kt
package com.example.trialgame

import android.content.Intent
import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import android.util.Log.d


import kotlinx.android.synthetic.main.activity_main.*
import kotlinx.android.synthetic.main.content_main.*



class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        setSupportActionBar(toolbar)

       nextpage.setOnClickListener {
           d("dalton", "button pressed")
           //d("dalton", "${editText.text}"
           //message.text = "${editText.text}"
           startActivity(Intent(this, AboutMe::class.java))
       }

    }



}
